.taskcat.yml

```diff
@@ -23,22 +24,23 @@ tests:
       Section1ToolingAndLoggingRegion: "us-east-2"
       Section1LogArchivingAccountId: ""
       Section1OrganizationRootID: ""
-      Section1DeliveryBucketPrefix: logs-delivery-$[taskcat_random-string]
+      Section1DeliveryBucketPrefix: logs-delivery
```

Comment:
previously I have seen that this fails without random string any reason why we are removing it.